### PR TITLE
Clarify 'Registration Form' project step 33 instructions

### LINKED ISSUE #46750: Unclear instructions for Registration Form Step 33
### Describe the Issue

I've encountered this several times now in the forums. The test will only pass if the `textarea` is added at the end of the `fieldset` after all other inputs, but the instructions do not make this clear. Thus, campers are adding it at the beginning of the `fieldset` and although their HTML is correct, the test is not passing because they have added it in the wrong spot.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-33

### Your code

N/A

### Expected behavior

Either clarify in the instructions that the `textarea` must be added at the bottom of the `fieldset` after all other inputs or remove the restriction as to where it is added.

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Ssevve**
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46750

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60facde2d0dc61085b41063f.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ dashedName: step-33
 
 The `textarea` element acts like an `input` element of type `text`, but comes with the added benefit of being able to receive multi-line text, and an initial number of text rows and columns.
 
-To allow users to register with a bio, add a `label` with the text `Provide a bio:` followed by a `textarea` element, which requires a closing tag.
+To allow users to register with a bio, at the end of the `fieldset` add a `label` with the text `Provide a bio:` followed by a `textarea` element, which requires a closing tag.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60facde2d0dc61085b41063f.md:12]: This is a monster of a sentence. I think we need simpler sentences. Perhaps:
```suggestion
Users will be able to register with a bio. Add a `label` with the text `Provide a bio:` at the end of the `fieldset`. Add a `textarea` element after the `label` element. Note that the `textarea` requires a closing tag.
```

Comment (Ssevve) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60facde2d0dc61085b41063f.md:12]: Sounds great! Your changes look a lot better.

Comment (Ssevve) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60facde2d0dc61085b41063f.md:12]: Sounds great! Your changes look a lot better.

**Review by jeremylt — APPROVED** [review]
I like it, thanks! Let's see what everyone else thinks.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**